### PR TITLE
Admin severe log alert: No line breaks in stack trace #4759

### LINKED ISSUE #4759: Admin severe log alert: No line breaks in stack trace
Stack traces in the periodic severe log alert emails look like this. Seems the line breaks in the stack trace have been lost.
image

### PR DESCRIPTION
**Pull request by thyageshm**
fixes #4759

### CODE REVIEW

**Comment by thyageshm** [comment]
@wkurniawan07 for your review

File: src/main/java/teammates/storage/search/SearchManager.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ private static boolean tryPutDocument(String indexName, Document document){
             //if it's a transient error in the server, it can be re-tried
             if(!StatusCode.TRANSIENT_ERROR.equals(e.getOperationResult().getCode())){
                 log.severe(String.format(ERROR_NON_TRANSIENT_BACKEND_ISSUE, document, indexName) 
-                        + " e:\n" + e.getStackTrace());
+                        + " e:\n" + TeammatesException.toStringWithStackTrace(e));
```
Comment (wkurniawan07) [inline src/main/java/teammates/storage/search/SearchManager.java]: `\n` just just next to it can be replaced as well.

**Comment by wkurniawan07** [comment]
Minor issue, good to go otherwise :+1:

**Comment by thyageshm** [comment]
done!

**Comment by wkurniawan07** [comment]
Looks ok, but just to make sure are you going to target all those severe logs that don't use `TeammatesException.toStringWithStackTrace` in this PR as well? If yes I'm pretty sure there's a handful more out there.

**Comment by thyageshm** [comment]
I have made the necessary changes to the files. Now I replace \n with br in the email generation stage so that we dont have to change many pages. Also, I am not sure if the log viewer on the admin side would have to be changed if we update \n to br everywhere

**Comment by wkurniawan07** [comment]
Fair enough, that can be another issue I guess.

**Comment by thyageshm** [comment]
I mean, I retained the \n because I dont know if <br> would break anything. Now that we are continuing to use \n, there should be no problems :-)

**Pull request merged**